### ISSUE TITLE
Add hot key extension point

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#1865]
**Considerations**
- Looks like the extension API affordance is for hotkeys in the manifest: https://developer.chrome.com/docs/extensions/reference/commands/
- We might want to register document handlers at document begin instead of document idle so our handler can take precedence over any defined on the page (see #402, where we moved out modal to an iframe to avoid host hotkeys)

**Comment by fregante** [comment]
I may be wrong but I think that the chrome.commands API can't be used here as they're meant to be global and static (i.e. defined in the manifest and customizable via Chrome's UI only)

If you want to register a hotkey in the current page you'll probably have to listen to keyboard events manually, via regular keydown event handlers, in the content script.

**Comment by twschiller** [comment]
> I may be wrong but I think that the chrome.commands API can't be used here as they're meant to be global and static (i.e. defined in the manifest and customizable via Chrome's UI only)

Correct, I think the manifest hotkeys are static, so we'd need to handle them ourselves. For #1866 we can register a hotkey which will then give PixieBrix access to the page

**Comment by twschiller** [comment]
Closing - will re-visit if custom hotkeys are placed on the roadmap
